### DISCUSSION TITLE
API developer donation links

### DISCUSSION THREAD

**Discussion by lachlan-00** [ampache/ampache#4012] [General]
I've taken donations to me and the project as i've been working a lot more in a new job and don't have the full time availability i used to have.

I would like to create a page/links in the www site and readme to direct people downstream who develop things.

people like @mitchray @lusum @Andrew-McGee @icefields who do a lot of work and don't get a lot of recognition.

If you want to add github sponsorship, paypal, other donation links here i'd really like to direct people to your projects and then give them as users the opportunity to donate to you instead of me.

**Comment by icefields** [comment]
Appreciate the mention. I do have a couple of links I can share here, for donating to the development of Power Ampache 2. 

<table>
  <tr>
    <th>
      <a target="_blank" href="https://www.patreon.com/Icefields">
      <img height="50" hspace="20" alt="Become a patreon" src="https://github.com/user-attachments/assets/3318ab05-3c7e-42dd-8784-f12129c0915d"></a>
    </th>
    <th>
      <a target="_blank" href="https://live.blockcypher.com/btc/address/bc1qm9dvdrukgrqpg5f7466u4cy7tfvwcsc8pqshl4">
      <img height="50" hspace="20" alt="Donate Bitcoin" src="https://power.ampache.dev/images/banner_bitcoin.png"></a>
    </th>
    <th>
      <a target="_blank" href="https://paypal.me/powerampache">
      <img height="50" hspace="20" alt="Donate - Paypal" src="https://power.ampache.dev/images/banner_paypal.png"></a>
    </th>
    </tr>
</table>

Patreon
https://www.patreon.com/Icefields

Paypal
https://paypal.me/powerampache

Bitcoin
bitcoin:bc1qm9dvdrukgrqpg5f7466u4cy7tfvwcsc8pqshl4

**Comment by Andrew-McGee** [comment]
Thanks Lachlan. Like you I am quite time poor due to work commitments so for me this is a passion/hobby based on best effort. No recognition besides 'hi' is needed! :)  Currently revisiting and fixing/finishing my front end. I would like to help out more with the ampache code base but tbh I find it a bit overwhelming. Do you have any doco or notes you are able to share that could help me get my head around it?

**Reply by lachlan-00** [reply]
depending on the area I can definitely tell and write things up for different areas.

I really need to document more things as there are only a few of us deeply familiar with the repo

**Comment by mitchray** [comment]
❤️ https://buymeacoffee.com/mitchray

**Comment by lachlan-00** [comment]
@fufroma does dogmazic have a donation link too?

**Reply by fufroma** [reply]
That's very kind of you to list people :) ! We have https://paypal.me/musiquelibre

**Comment by lachlan-00** [comment]
https://ampache.org/donate.html points to the readme for now
![image](https://github.com/user-attachments/assets/c2131c5c-d753-46b7-94f3-2f10b66c1821)


which points to https://github.com/ampache/ampache?tab=readme-ov-file#credits

Happy to add anyone and everyone here. / also any links.

I'll also advertise on other places

**Comment by icefields** [comment]
I finally finished my patreon account setup:
https://www.patreon.com/Icefields

**Reply by lachlan-00** [reply]
do they have a badge logo like the others?

**Reply by icefields** [reply]
yes, I just upadated my initial post.

**Reply by icefields** [reply]
My designer also gave me custom badges she designed, for Paypal and Bitcoin. The ones before were from FDroid.

**Reply by lachlan-00** [reply]
i've added patreon but feel free to edit/pull yourself for any other changes
https://github.com/ampache/ampache/blob/develop/README.md
